libsvm-wasm.c

```diff
@@ -17,22 +17,26 @@ typedef struct svm_parameter svm_parameter;
 void print_null(const char *s) {}
 
 EMSCRIPTEN_KEEPALIVE
-svm_node* init_node(double* data, int size) {
-    svm_node* node = MALLOC(svm_node, size+1);
-    for (int i = 0; i < size; i++) {
+svm_node *init_node(double *data, int size)
+{
+    svm_node *node = MALLOC(svm_node, size + 1);
+    for (int i = 0; i < size; i++)
+    {
         node[i].index = i + 1;
         node[i].value = data[i];
     }
     node[size].index = -1;
     return node;
 }
 
-// EMSCRIPTEN_KEEPALIVE
-// bool save_model(svm_model* model, char* name) {
-//     int success = svm_save_model(name, model);
-//     if (success < 0) return false;
-//     return true;
-// }
+EMSCRIPTEN_KEEPALIVE
+bool save_model(svm_model *model, const char *model_file_name)
+{
+    int success = svm_save_model(model, model_file_name);
+    if (success < 0)
+        return false;
+    return true;
+}
```

Comment:
This seems to be different from what you have commented in the issue.

Reply:
it's thesame
```c
bool save_model(svm_model *model, const char *model_file_name)
{
    int success = svm_save_model(model, model_file_name);
    if (success < 0)
        return false;
    return true;
}
```

Reply:
can you make this more clear, I don't quite understand what you mean

Reply:
Humm I get it! You implemented `svm_save_model` for wasm in the `libsvm/libsvm.cpp`. I think we should implement it under `libsvm-wasm.c`.

What's your insight about it?

Reply:
I've made some new commit. The full implementation is now moved into `lib-wasm.c` also I've been able to implement the `load_model` and `evaluation` method